app/lib/schema_file_loader.rb

```diff
@@ -74,9 +74,9 @@ def prepare_directories(dest_dir)
 
     def get_missing_downloads(dest_dir)
       download_files = EFILE_SCHEMAS_FILENAMES.map do |(filename, download_folder, optional)|
-        [File.join(dest_dir, download_folder, filename), optional]
+        [File.join(dest_dir, download_folder, filename), download_folder, optional]
       end
-      download_files.filter do |(download_file, optional)|
+      download_files.filter do |(download_file, _download_folder, _optional)|
```

Comment:
is `_optional` here redundant? idk

Reply:
looks like you can just use `_` for both unused params